src/field_theory/polynomial_galois_group.lean

```diff
@@ -415,7 +415,35 @@ begin
   { rw ← equiv.perm.card_support_eq_two,
     apply nat.add_left_cancel,
     rw [←p_roots, ←set.to_finset_card (root_set p ℝ), ←set.to_finset_card (root_set p ℂ)],
-    exact gal_action_hom_bijective_of_prime_degree_aux.symm },
+    exact (gal_action_hom_bijective_of_prime_degree_aux p).symm },
+end
+
+/-- Same as above, but with extra flexibility -/
+lemma gal_action_hom_bijective_of_prime_degree'
```

Comment:
Does one of the two lemmas easily imply the other? Is there a common generalization? Currently both proofs are "longish".

Reply:
The first lemma is used in the proof of the second lemma.